agent/node.go

```diff
@@ -74,6 +74,10 @@ type NodeConfig struct {
 	// HeartbeatTick defines the amount of ticks between each
 	// heartbeat sent to other members for health-check purposes
 	HeartbeatTick uint32
+
+	// Availability allows a user to control the current scheduling
+	// status of a node.
```

Comment:
Thanks @aaronlehmann I forgot to change the comment in the last update. Sorry about that. The comment has been updated.